bgpd/bgp_updgrp.c

```diff
@@ -1182,6 +1182,8 @@ static void update_subgroup_copy_adj_out(struct update_subgroup *source,
 		aout_copy->attr =
 			aout->attr ? bgp_attr_intern(aout->attr) : NULL;
 	}
+
+	dest->scount = source->scount;
```

Comment:
What is this? This looks like a bug fix independent of json changes.  If so get it into it's own commit with it's own comment please.

Reply:
Sure, I will separate this commit.